99-Swiss officer ratios.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>Please add a couple comments to the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>